hermes/README.md

```diff
@@ -0,0 +1,200 @@
+        ' | PythNet Node 3 | --- | PythNet Node 4 | '
+        ' +----------------+     +----------------+ '
+        '         |Hermes|               |Hermes|   '
+        '         +------+               +------+   '
+        + - - - - - - - - - - - - - - - - - - - - - +
+                            ^
+                            |
+                            v
+                 +---------------------+
+                 |      Wormhole       |
+                 +---------------------+
+```
+
+
+In this new design, the Price Service is integrated into the Hermes node
+service, decentralizing the API. Hermes is now also responsible for direct
+communication with Wormhole over P2P, which reduces latency and simplifies
+responsibilities.
+
+The new design offers several benefits:
+
+1. Hermes can participate as a P2P node in the Wormhole network directly.
+2. Hermes nodes form a Pyth-specific P2P network with fast communication.
+3. Hermes can directly observe on-chain state for faster operation.
+4. Hermes can have its identity tied to a PythNet node for authenticated operation.
+5. Data ownership is clearer with the removal of the middleman.
+
+With tighter communication flow, we can define new behaviors such as
+Pyth-specific threshold signing, fast price accumulation with proving (due to
+direct node access), improved metrics and observations, and the ability for
+users to run observe-only Hermes nodes to watch the Pyth network directly
+instead of relying on a Price Service host.
+
+The Hermes node architecture is as follows:
+
+---
+
+![image](https://user-images.githubusercontent.com/158967/225939587-f19cfe77-0393-4798-ad72-0022420d3e51.png)
+
+---
+
+This is more in line with other well-known projects in the blockchain and P2P
+space, such as Solana, Bitcoin, and Cosmos chains, where users who run the node
+can also act as observers with specific features disabled.
+
+In our case, the node software can be run with or without a PythNet validator
+running due to it being designed to fetch updates from either Pyth or Geyser
+networks for high availability.
+
+## Components
+
+The Hermes Node software consists of the following components:
+
+- **Wormhole P2P Connector**: Connects to the Wormhole P2P network and listens for VAAs.
+- **Pyth P2P Connector**: Connects to the Pyth P2P network and listens for Account Updates.
+- **Geyser Connector**: Connects to the Geyser node software and listens for Account Updates.
+- **REST API**: Provides an interface for external applications to interact with PythNet.
+- **Websocket API**: Offers real-time data streaming for interacting with PythNet.
+
+While Hermes will always participate in the Wormhole and Pyth P2P networks, the
+Pyth network shares network updates on the Pyth layer and so can be run without
+a PythNet node running along-side it for a spy-only mode. This can be done by
+running without specifying `--geyser-socket`.
+
+## Getting Started
```

Comment:
suggest putting this section before the architecture overview, as it's more likely that the reader wants this than the architecture